internal/locate/region_cache.go

```diff
@@ -371,7 +371,7 @@ type RegionCache struct {
 		sync.RWMutex                           // mutex protect cached region
 		regions        map[RegionVerID]*Region // cached regions are organized as regionVerID to region ref mapping
 		latestVersions map[uint64]RegionVerID  // cache the map from regionID to its latest RegionVerID
-		sorted         *btree.BTree            // cache regions are organized as sorted key to region ref mapping
+		sorted         *Sorted                 // cache regions are organized as sorted key to region ref mapping
```

Comment:
Let's call this type `SortedRegions`. It's a specialization of the generic `BTree[Region]`.

Reply:
OK, I have changed it.